app/models/benchmark_result.rb

```diff
@@ -22,7 +22,7 @@
 #
 
 class BenchmarkResult < ApplicationRecord
-  belongs_to :benchmark_result_school_generation_run
+  belongs_to :benchmark_result_school_generation_run, counter_cache: :benchmark_result_count
```

Comment:
Are we ok with counter cache in 2020? It could just be a column we update when we finish, but it's a bit neater doing in this way (although almost certainly slower)

Reply:
As long as we make all changes to the thing that's being cached via AR then it does the job.